### DISCUSSION TITLE
The Extension Manifest is Invalid

### DISCUSSION THREAD

**Discussion by JimKay1941** [Dotnet-Boxed/Templates#143] [Q&A]
I downloaded this extension but I cannot install it.

The error message is the title.

**Comment by RehanSaeed** [comment]
What version of VS are you using?

**Comment by JimKay1941** [comment]
VS 2015 community. All updates applied.

**Comment by RehanSaeed** [comment]
Just tested and the install seems ok. Was your download corrupted?

**Comment by JimKay1941** [comment]
Thanks for the reply,
I tried it all again (including a rather large Nuget update, and it all seems to be OK now.

**Comment by RehanSaeed** [comment]
Cool, thanks for checking!
